src/plugins/intel_cpu/src/nodes/kernels/x64/registers_pool.hpp

```diff
@@ -52,10 +52,16 @@ class RegistersPool {
             initialize(regPool, requestedIdx);
         }
         ~Reg() {
-            release();
+            try {
```

Comment:
Instead try catch it would be better to have release() version which not throw as exception usually should not pop in dtors.
Is it possible that it will throw or Coverity issue is because this function may throw but is not possible in dtor?

Reply:
In this particular scenario, the exception can be caught as intended. The mechanism seems to heavily dependent on exception throwing

Reply:
> Is it possible that it will throw or Coverity issue is because this function may throw but is not possible in dtor?

Not here. Coverity complains, because dtor should not throw. In that case ov::Exception is handled inside, no there is no throw that will unwind outside the dtor that could cause `std::terminate`